uyi-FW/StackOverflow-lite
# #183365886 Project Setup

#### What does this PR do?
This configures the MySQl database using Sequelize

#### Description of Task to be completed?
Start server on [localhost](http://localhost:3000) and connect to the db

#### How should this be manually tested?
- After cloning the repo, `cd` into it and run `npm i` to install the dependencies.
- Start the server by running `npm run dev`
- test this on your browser by visiting http://localhost:3000. Your browser should display
 > {"message":"hello from api"}

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#183365886](https://www.pivotaltracker.com/story/show/183365886)

#### Screenshots (if appropriate)
None

#### Questions:
None
